ansible/roles/schulcloud-server-core/tasks/main.yml

```diff
@@ -175,6 +175,15 @@
     tags:
       - ingress
 
+  - name: tldaraw migration Job
+    kubernetes.core.k8s:
+      kubeconfig: ~/.kube/config
+      namespace: "{{ NAMESPACE }}"
+      template: tldraw-migration-job.yml.j2
+      state: "{{ 'present' if WITH_TLDRAW2 else 'absent'}}"
```

Comment:
Was state macht ist mir nicht wirkich klar.

Reply:
Erstellt/Deployt ein job wenn WITH_TLDRAW2 true ist, sonst nicht.
Und dann wird ausgeführt in dem Release wo wir WITH_TLDRAW2 auf true setzen

Reply:
https://docs.ansible.com/ansible/latest/collections/kubernetes/core/k8s_module.html#parameter-state

Reply:
Super danke. :)